doctr/transforms/modules/base.py

```diff
@@ -89,21 +90,45 @@ def __call__(self, img: Any) -> Any:
 
 
 class RandomRotate(NestedObject):
-    """Randomly rotate a tensor image
+    """Randomly rotate a tensor image and its boxes
 
     Args:
         max_angle: maximum angle for rotation, in degrees. Angles will be uniformly picked in
             [-max_angle, max_angle]
         expand: whether the image should be padded before the rotation
     """
-    def __init__(self, max_angle: float = 25., expand: bool = False) -> None:
+    def __init__(self, max_angle: float = 5., expand: bool = True) -> None:
```

Comment:
While #442 isn't dealt with, do you think we should switch to `expand=True` ?
- advantages: we don't lose information
- inconvenient: aspect ratio is messed up

Since the resizing would be done afterwards, it's possible. But we have to understand the consequences